jsonrpclib/SimpleJSONRPCServer.py

```diff
@@ -164,12 +170,14 @@ def do_POST(self):
             L = []
             while size_remaining:
                 chunk_size = min(size_remaining, max_chunk_size)
-                L.append(self.rfile.read(chunk_size))
+                chunk = self.rfile.read(chunk_size).decode()
```

Comment:
Hm, this is a tricky one -- I need to look at this in the larger context, but what if this is UTF8, etc?